xonsh/completers/completer.py

```diff
@@ -8,7 +8,13 @@ def complete_completer(prefix, line, start, end, ctx):
     args = line.split(" ")
     if len(args) == 0 or args[0] != "completer":
         return None
-    curix = args.index(prefix)
+    try:
+        curix = args.index(prefix)
+    except ValueError:
```

Comment:
Rather than using index() &  Value error, it is probably best do something like:

```
curix = next(i for i, x in enumerate(args) if x == prefix, None)
if curix is None:
    return None
```

Reply:
I am pretty sure that this is more performant than try-except

Reply:
Did it even more performantly 🐳 
What do you think?